src/views/Product/Page.tsx

```diff
@@ -118,19 +160,13 @@ class Page extends React.PureComponent<{ product: ProductDetails_product }> {
                       className="product-page__product__gallery"
                       ref={this.productGallery}
                     >
-                      {product.images.map((image, index) => (
-                        <CachedImage url={image.url} key={image.id}>
-                          <Thumbnail source={product} />
-                        </CachedImage>
-                      ))}
+                      {this.renderImages(product)}
                     </div>
                     <div className="product-page__product__info">
                       <div
                         className={classNames({
-                          ["product-page__product__info--fixed"]: this
-                            .showCarousel,
+                          ["product-page__product__info--fixed"]: true,
```

Comment:
Might be simplified to `className="product-page__product__info--fixed"`

Reply:
Fixed